frontend/src/routes/user/login@user.svelte

```diff
@@ -20,49 +18,45 @@
 
 	async function login(): Promise<void> {
 		try {
-			await UserService.login({
-				requestBody: {
-					email: email,
-					password
-				}
-			})
+			const requestBody = {
+				email,
+				password
+			}
+
+			await UserService.login({ requestBody })
+			// Once logged in, we can fetch the workspaces
+			$usersWorkspaceStore = await WorkspaceService.listUserWorkspaces()
```

Comment:
Once logged in is not the only scenario where you want to fetch the workspaces. For instance, if you were already logged and resumed your session from a past token. In the previous logic, you would load it in the relevant layout if it wasn't set: https://github.com/windmill-labs/windmill/pull/65/files#diff-68f6c24b32d9449d271d5c2384430f3b9185e432dcd15aba6d6d6ba494efc22eL85
This is essentially what you want